server/services/index.js

```diff
@@ -2,6 +2,7 @@ module.exports.example = require('./example');
 module.exports.caldav = require('./caldav');
 module.exports.openweather = require('./openweather');
 module.exports.mqtt = require('./mqtt');
+module.exports.xiaomi = require('./owntracks');
```

Comment:
wow be careful you are overriding the xiaomi integration

Reply:
this is weird, I just checked and I do not have this in my code ... is this possible something is wrong in diff ? 
https://github.com/ngeissel/Gladys/blob/Handle_Owntracks_MQTT_messages/server/services/index.js